File: 9C-1.py
def high_and_low(str):
    array = list(map(int, str.split()))
    return f"{max(array)} {min(array)}"


print (high_and_low("1 2 3 4 5"))
print (high_and_low("1 2 -3 4 5"))
print (high_and_low("1 9 3 4 -5"))

File: 9C-1.js
function highAndLow(str) {
  let array = str.split(" ").map(Number);
  let max = Math.max.apply(null, array);
  let min = Math.min.apply(null, array);
  return `${max} ${min}`;
}

console.log(highAndLow("1 2 3 4 5"));
console.log(highAndLow("1 2 -3 4 5"));
console.log(highAndLow("1 9 3 4 -5"));
